multicluster/build/yamls/antrea-multicluster-leader-namespaced.yml

```diff
@@ -6,6 +6,12 @@ metadata:
   name: antrea-multicluster-controller
   namespace: changeme
 ---
+apiVersion: v1
```

Comment:
Do we need this? Because we need one for each member, I dint want to create this in the manifest by default?